00-programming-fundamentals/code-coverage-and-simplecov.md

```diff
@@ -102,7 +102,7 @@ Some teams require a certain percentage of code coverage and it's likely you wil
 
 ### Exercise
 
-Add code coverage to your Alouette Weekend-Warrior exercise.      
+Add code coverage to your Robot Name Weekend-Warrior exercise.      
```

Comment:
Should we change it to "your latest Weekend-Warrior exercise" since they haven't yet done Robot Name either?

Reply:
Since we now have my belated Regex exercise we could do them in the given order and Regex in week 3, or do regex later and do hamming 1st.